arm-servicebus/2015-08-01/swagger/servicebus.json

```diff
@@ -1586,12 +1699,12 @@
       "x-ms-azure-resource": true,
       "description": "The Resource definition for other than namespace."
     },
-    "NamespaceListResult": {
+    "SBNamespaceListResult": {
```

Comment:
Yes, I believe this would introduce breaking changes. While introducing newer services you can probably go ahead and use the new naming convention, I'm not sure if you want to do this to the sdks that have been already published. @anychandy are there any guidelines regarding renaming existing "Namespace" model definitions?